### PR TITLE
Bug ast pivot points

### PR DESCRIPTION
**Pull request by vEpiphyte**
a few more places to sleep inside of the ast and storm commands

### CODE REVIEW

File: synapse/lib/ast.py [hunk 1]
```diff
@@ -2064,7 +2081,7 @@ async def cond(node, path):
             prop = node.form.props.get(name)
             if prop is None:
                 mesg = f'No property named {name}.'
-                raise s_exc.NoSuchProp(name=name)
+                raise s_exc.NoSuchProp(mesg=mesg, name=name)
```
Comment (jnwatson) [inline synapse/lib/ast.py:2084]: lol

File: synapse/lib/ast.py [hunk 2]
```diff
@@ -1372,6 +1386,9 @@ async def filter(x, path):
             for name, _ in node.getTags(leaf=leaf):
 
                 if not await filter(name, path):
+                    # adding this for strawman discussion. We are at worst case in a
+                    # terrible compute method which sucks to evaluate each time through.
+                    await asyncio.sleep(0)
```
Comment (jnwatson) [inline synapse/lib/ast.py:1391]: yeah we need this

**Review by jnwatson — DISMISSED** [review]
Probably want to remove 1 comment.

File: synapse/lib/ast.py (hunk 2 above)
Comment (invisig0th) [reply synapse/lib/ast.py:1391]: omg yes

**Review by invisig0th — APPROVED** [review]

**Pull request merged**